cmake/Install.cmake

```diff
@@ -136,22 +136,16 @@ install_remote(FILES "https://raw.github.com/NREL/EnergyPlusBuildSupport/v8.2.0/
 
 # TODO Remove version from file name or generate
 # These files names are stored in variables because they also appear as start menu shortcuts later.
```

Comment:
@kbenne It says here these variables were being used as start menu shortcuts, but I didn't see that.  I'm hoping to clean up the install a little and get rid of some of these files (since we are more public now).  Can you confirm whether I broke something here or not? 

For the record, I was able to ```make package``` with this branch on my local machine and install it (well, [almost](https://github.com/NREL/EnergyPlus/issues/4781)) and the changelog had been created and installed nicely.  

I'm open to changing the scripts to create html instead of md, csv, etc. for the installation packages.